src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Publish.targets

```diff
@@ -710,6 +710,10 @@ Copyright (c) .NET Foundation. All rights reserved.
           DependsOnTargets="_ComputeExcludeFromPublishPackageReferences;
                             _ParseTargetManifestFiles">
 
+    <PropertyGroup>
+      <PublishDepsFilePath Condition=" '$(PublishDepsFilePath)' == '' ">$(PublishDir)$(ProjectDepsFileName)</PublishDepsFilePath>
```

Comment:
I think this should move into the `PropertyGroup` below after `_UseBuildDependencyFile ` is set and be conditioned on `'$(_UseBuildDependencyFile )' != 'true'`.

Reply:
Actually, scratch that change for a second.  Let me rethink this.